src/main/java/mate/academy/springboot/swagger/util/PageRequestUtil.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+
+public class PageRequestUtil {
+    private static final String SEPARATOR_ATTRIBUTES = ":";
+    private static final String SEPARATOR_PARAMS = ";";
+    private static final int INDEX_FIELD_NAME = 0;
+    private static final int INDEX_DIRECTION_VALUE = 1;
+
+    public static PageRequest getPageRequest(Integer page, Integer count, String sortBy) {
```

Comment:
```suggestion
    public PageRequest getPageRequest(Integer page, Integer count, String sortBy) {
```
don't use static if it's possible, it's a bad practice